project/ws/app/src/lib/routes/org/components/org/org.component.html

```diff
@@ -7,17 +7,20 @@
         <div class="user-avatar ">
           <div class="flex mt-10">
             <div class="flex-auto mr-4">
-              <div class="image-outer">
+              <div class="image-outer" *ngIf="currentOrgData?.sourceName !== 'WHO'">
```

Comment:
@revathip  Please remove this org specific logic.

Reply:
Done